exercises/perfect-numbers/example.rb

```diff
@@ -0,0 +1,39 @@
+class PerfectNumber
+  VERSION = 1
+  attr_accessor :num
+  
+  def initialize(num)
+    raise "#{num} is not a natural number." if num < 0
```

Comment:
Usually I write my exception messages without relying on variables (the exception being if it is a custom Exception). In this case we could use a generic message like  `a natural number is expected.` or something similar. What do you think?